x-pack/filebeat/module/aws/elb/test/example-alb-http.log-expected.json

```diff
@@ -368,5 +368,220 @@
+        "aws.elb.backend.ip": "10.0.0.1",
+        "aws.elb.backend.port": "80",
+        "aws.elb.backend_processing_time.sec": 0.048,
+        "aws.elb.chosen_cert.arn": "arn:aws:acm:us-east-2:123456789012:certificate/12345678-1234-1234-1234-123456789012",
+        "aws.elb.matched_rule_priority": "1",
+        "aws.elb.name": "app/my-loadbalancer/50dc6c495c0c9188",
+        "aws.elb.protocol": "http",
+        "aws.elb.request_processing_time.sec": 0.086,
+        "aws.elb.response_processing_time.sec": 0.037,
+        "aws.elb.ssl_cipher": "ECDHE-RSA-AES128-GCM-SHA256",
+        "aws.elb.ssl_protocol": "TLSv1.2",
+        "aws.elb.target_group.arn": "arn:aws:elasticloadbalancing:us-east-2:123456789012:targetgroup/my-targets/73e2d6bc24d8a067",
+        "aws.elb.target_port": [
+            "10.0.0.1:80"
+        ],
+        "aws.elb.target_status_code": [
+            "200"
+        ],
+        "aws.elb.trace_id": "Root=1-58337281-1d84f3d73c47ec4e58577259",
+        "aws.elb.type": "https",
+        "cloud.provider": "aws",
+        "destination.domain": "www.example.com",
+        "event.category": "web",
+        "event.dataset": "aws.elb",
+        "event.end": "2018-07-02T22:23:00.186Z",
+        "event.kind": "event",
+        "event.module": "aws",
+        "event.outcome": "success",
+        "event.start": "2018-07-02T22:22:48.364000Z",
+        "fileset.name": "elb",
+        "http.request.body.bytes": 0,
+        "http.request.method": "GET",
+        "http.request.referrer": "https://www.example.com:443/",
+        "http.response.body.bytes": 57,
+        "http.response.status_code": 200,
+        "http.version": "1.1",
+        "input.type": "log",
+        "log.offset": 3750,
+        "service.type": "aws",
+        "source.ip": "192.168.131.39",
+        "source.port": "2817",
+        "tags": [
+            "forwarded"
+        ],
+        "tls.cipher": "ECDHE-RSA-AES128-GCM-SHA256",
+        "tls.version": "1.2",
+        "tls.version_protocol": "tls",
+        "tracing.trace.id": "Root=1-58337281-1d84f3d73c47ec4e58577259",
+        "user_agent.original": "curl/7.46.0"
+    },
+    {
+        "@timestamp": "2018-07-02T22:23:00.186Z",
+        "aws.elb.action_executed": [
+            "forward"
+        ],
+        "aws.elb.backend.http.response.status_code": 101,
+        "aws.elb.backend.ip": "10.0.1.192",
+        "aws.elb.backend.port": "8010",
+        "aws.elb.backend_processing_time.sec": 0.003,
+        "aws.elb.matched_rule_priority": "1",
+        "aws.elb.name": "app/my-loadbalancer/50dc6c495c0c9188",
+        "aws.elb.protocol": "http",
+        "aws.elb.request_processing_time.sec": 0.001,
+        "aws.elb.response_processing_time.sec": 0.0,
+        "aws.elb.target_group.arn": "arn:aws:elasticloadbalancing:us-east-2:123456789012:targetgroup/my-targets/73e2d6bc24d8a067",
+        "aws.elb.target_port": [
+            "10.0.1.192:8010"
+        ],
+        "aws.elb.target_status_code": [
+            "101"
```

Comment:
I guess there's a good reason to not use integers for the status code :sweat_smile:

Reply:
Seems like in order to use the split processor later, the field has to be string with a separater. `Splits a field into an array using a separator character. Only works on string fields.`